db/migrate/20231123233308_delete_duplicated_holidays.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class DeleteDuplicatedHolidays < ActiveRecord::Migration[7.0]
+  def up
+    execute <<~SQL
+      DELETE
+        FROM calendar_events ce
+      WHERE
+        ce.id IN (SELECT ce2.id FROM calendar_events ce2
+                  INNER JOIN users ON users.id = ce2.user_id
+                  WHERE ce2.post_id IS NULL
+                    AND ce2.username != users.username)
```

Comment:
Do we have to worry about case insensitivity here?

Reply:
I think it is ok. Right now, if someone change username from `kris` to `Kris`, we would have duplicated holiday. We still want to delete `kris` and display properly `Kris`